client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,12 +97,26 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
+			const moment = this.props.moment;
+			const now = moment();
+			const scheduledDate = moment( this.props.post.date );
+			// If the content is scheduled to be release within a year, do not display the year at the end
+			const scheduledTime = scheduledDate.calendar( null, {
+				sameElse: this.props.translate( 'll [at] LT', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'LL [at] LT' )` *ES Score: 12*